### PR TITLE
Added JSDOC for initDismissibles for #9495

### PR DESCRIPTION
**Pull request by Yekasumah**
I have added the Jsdoc to the initDismissibles for issue #9495. I haven't completed the tests yet.
I'll be updating the PR as I continue to make changes.

### CODE REVIEW

File: client/src/components/CommentApp/main.scss [hunk 1]
```diff
@@ -62,7 +62,9 @@ $box-padding: 10px;
     width: calc(100% + #{$box-padding} * 2);
     padding: 5px 10px;
     box-sizing: border-box;
-
+    border-bottom-left-radius: $box-border-radius;
+    border-bottom-right-radius: $box-border-radius;
+
```
Comment (lb-) [inline client/src/components/CommentApp/main.scss:67]: I think this change is not meant to be part of this PR.

**Comment by Yekasumah** [comment]
Hi @lb-, I'm sorry about that didn't know the commit was attached to the branch. I tried creating a new branch for it, but that commit is still going with it even after reverting the changes. Please is there anything I can do?

**Comment by lb-** [comment]
You will need to make a new branch most likely, otherwise you will keep having this issue.

Best to create a new branch from your local main, something like `fix/9495-init-dismissibles-js`.

You can do this via the command line with
`git checkout -b fix/9495-init-dismissibles-js`

While on that branch, make a new commit that reverses/undoes the changes to the CSS.

Now push that branch up to your GitHub account.

Then open Wagtail on Github and you will see that you can create a new PR with it.

**Comment by lb-** [comment]
You may need to close this PR as it is linked to your `main` branch sorry.

**Pull request closed without merging**